CMakeLists.txt

```diff
@@ -121,13 +134,15 @@ endif ( EGOA_ENABLE_BOOST )
 # Google tests
 if ( EGOA_ENABLE_TESTS )
     include ( CTest )
-    if ( EGOA_TEST_FRAMEWORK STREQUAL "OfflineGoogleTestFramework" )
-        find_package ( GoogleTestFramework PATHS ${EGOA_TEST_FRAMEWORK_LOCATION})
-    elseif ( EGOA_TEST_FRAMEWORK STREQUAL "OnlineGoogleTestFramework" )
-        find_package ( GoogleTestFrameworkOnlineVersion )
-    else ()
-        find_package ( GoogleTestFramework )
-    endif ()
```

Comment:
This is not working for Windows and MacOS. Thus, we call the old cmake file that copes with the issue.